### PR TITLE
Update Collaboration README.md

### PR DESCRIPTION
**Pull request by Bikaze**
Add the group norms

### CODE REVIEW

File: collaboration/README.md [hunk 1]
```diff
@@ -1,5 +1,15 @@
 # Collaboration
 
-<!-- group norms summary -->
+## Group Norms
+
+Our group is committed to fostering a respectful and supportive environment where we honor each other's time and cultural differences, while providing flexible communication channels to accommodate diverse schedules.
```
Comment (Mo-Altayeb) [inline collaboration/README.md:5]: the summary should be under 'collaboration' and the norms under the 'group norms' header

**Review by ObayCipher — DISMISSED** [review]
Fabulous and can be edited for better at the future 
Summary under Collaboration
Norms under Group Norms

**Review by Mo-Altayeb — COMMENTED** [review]
Great! One final thing is you should delete the blank lines 14 and 15 to pass the CI check, then we can merge it

File: collaboration/README.md [hunk 2]
```diff
@@ -1,5 +1,15 @@
...
+## Group Norms
+
+- We will respect each other's time while remaining flexible and understanding of unforeseen delays.
+- We will create a supportive space that acknowledges cultural differences by asking and offering help and encouragement from/to teammates facing any challenges
+- We will provide various communication channels whether synchronous or asynchronous to accommodate the different availability times of group members.
+- We will provide constructive criticism while putting into consideration any unforeseen or surprising challenges and refrain from assigning blame.
+- If a breach of trust happens, we will evaluate the mistakes that led to the breach of trust, learn from them, and implement strategies to prevent similar issues in the future.
+- We will use discussions to explore ideas and debate to make decisions.
+
+
```
Comment (Mo-Altayeb) [inline collaboration/README.md:14]: delete lines 14 and 15

Comment (ObayCipher) [reply collaboration/README.md]: I delete it but it auto generated again

**Review by Mo-Altayeb — APPROVED** [review]
all set 💯

**Comment by Bikaze** [comment]
I re-opened the issue because there were still some unresolved CI checks. I have cleared all errors thanks @Mo-Altayeb and @ObayCipher for great collabo

**Pull request merged**